backends/dpdk/base_test.py

```diff
@@ -0,0 +1,1142 @@
+# This file was copied and then modified from the file
```

Comment:
There is a lot of redundancy here. We can likely make this a shared library for both BMv2 and DPDK.